projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-vc.ts

```diff
@@ -335,6 +346,16 @@ export function clickOnConflictDetected(attribute: string): void {
   clickOnConflictSolverLink(attribute, 'Conflict Detected - View Details Link');
 }
 
+/**
+ * Navigates to the conflict group that contains an attribute which is involved in a conflict and waits for the request to finish.
+ *
+ * @param attribute - Attribute name
+ */
+export function clickOnConflictDetectedAndWait(attribute: string): void {
```

Comment:
@Uli-Tiger, could you add via which link will be clicked to navigate?
Otherwise, both methods `clickOnViewInConfigurationAndWait` and  `clickOnConflictDetectedAndWait` have the same description.